components/dashboard/src/admin/Settings.tsx

```diff
@@ -5,15 +5,32 @@
  */
 
 import { useContext } from "react";
-import { InstallationAdminSettings } from "@gitpod/gitpod-protocol";
+import { TelemetryData, InstallationAdminSettings } from "@gitpod/gitpod-protocol";
 import { AdminContext } from "../admin-context";
 import CheckBox from "../components/CheckBox";
 import { PageWithSubMenu } from "../components/PageWithSubMenu";
 import { getGitpodService } from "../service/service";
 import { adminMenu } from "./admin-menu";
+import { useEffect, useState } from "react";
+import InfoBox from "../components/InfoBox";
+import { Redirect } from "react-router-dom";
+import { UserContext } from "../user-context";
 
 export default function Settings() {
     const { adminSettings, setAdminSettings } = useContext(AdminContext);
+    const [telemetryData, setTelemetryData] = useState<TelemetryData>();
+    const { user } = useContext(UserContext);
+
+    useEffect(() => {
+        (async () => {
+            const data = await getGitpodService().server.adminGetTelemetryData();
+            setTelemetryData(data)
+        })();
+    });
+
+    if (!user || !user?.rolesOrPermissions?.includes('admin')) {
```

Comment:
After this new gaurd, `admin/settings` does not open when the user isn't a admin like you'd expect!